documentation/code-generation.md

```diff
@@ -94,3 +94,80 @@ generator –r –v –l=temp –version=v11.2.0-beta C:/azure-rest-api-specs/sp
 ## Generate Azure SDK for Go profiles
 
 Take a look into the [profile generator documentation](https://github.com/Azure/azure-sdk-for-go/tree/main/eng/tools/profileBuilder)
+
+## Generate Go Mgmt SDK from Typespec
```

Comment:
Since this is the recommended way, we should move this to the top of the document.
@raych1  we shouldn't need section `Generate an Azure-SDK-for-Go service package` anymore, right?

Reply:
@maririos for data plane, the generated code by DPG needs manual post processing. I suggest calling out this approach for management plane only for now.

Reply:
for go, the following step is only available for mpg. so, it makes sense to call out this.